docs/slack.rst

```diff
@@ -174,54 +174,130 @@ Our goal with Write the Docs is to be **the most welcoming community on the inte
 People who are new to our community in particular should be assisted and welcomed.
 
 Our :doc:`/code-of-conduct` applies to all spaces associated with Write the Docs, which includes our Slack.
-In addition, we have a few Slack-specific guidelines:
-
-Social rules
-~~~~~~~~~~~~
-
-- **Use inclusive language** - Our community is multicultural, multiracial, multigendered, and generally diverse. When posting in channels, avoid using words such as "guys" to address a mixed group. Possible alternatives: "folks", "everyone", "documentarians".
-- **We're all documentarians** - The Write the Docs community is much more than a space for technical writers. We welcome everyone who wants to learn more about creating documentation, including developers, testers, customer support agents, technical writers, program/product managers, developer relations specialists, and more.
-- **Only participate when you can be constructive** - Try to be constructive in all that you do, and otherwise consider not participating in a conversation.
-- **Say hi** - The world is a better place with a flood of hello's.
-- **Don't be snarky** - We are all trying the best we can.
-- **Be careful with jokes** - You know that you're joking, but some people might take you seriously.
-- **If you don't have anything nice to say, don't say anything at all**.
+In addition, we have several Slack-specific guidelines:
+
+Social tips
+~~~~~~~~~~~
+
+To have good experiences on Slack, create the conversations you want to be a part of.
+You can help us do this by following these tips:
+
+- **Say hi.**
+  Don't forget to introduce yourself in ``#intros``.
+  If you've been away a while, reintroduce yourself.
+  The world is a better place with a flood of hello's.
+- **Use inclusive language.**
+  Our community is multicultural, multiracial, multigendered, and generally diverse.
+  When posting in channels, avoid using words such as "guys" to address a mixed group; possible alternatives include "folks", "everyone", and "documentarians".
+- **Remember that we're all documentarians.**
+  The Write the Docs community is much more than a space for technical writers.
+  We welcome everyone who wants to learn more about creating documentation, including developers, testers, customer support agents, technical writers, program/product managers, developer relations specialists, and more.
```

Comment:
Shall we add a "don't diss any of these groups" sentence here?

Reply:
I liked the suggestion to act on this further down. See https://github.com/writethedocs/www/pull/1872/files#r1067046560 and 65b5bc69